src/navigation/bottom-tab.tsx

```diff
@@ -40,6 +44,10 @@ const CartIcon = ({ focused, color, size }: TabBarIconProps) => (
     <AntDesign name={focused ? 'shoppingcart' : 'shoppingcart'} color={color} size={size} />
 );
 
+const MapIcon = ({ focused, color, size }: TabBarIconProps) => (
+    <Entypo name={focused ? 'map' : 'map'} color={color} size={size} />
```

Comment:
For consistent styling, lets try and stick with MaterialIcons or MaterialCommunityIcons for now